.NET/Microsoft.Recognizers.Definitions/Microsoft.Recognizers.Definitions.csproj

```diff
@@ -12,7 +12,17 @@
     <GenerateSerializationAssemblies>Auto</GenerateSerializationAssemblies>
 	  <CodeAnalysisRuleSet>../Recognizers-Text.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
+<PropertyGroup>
+	  <!--
+		Make sure any documentation comments which are included in code get checked for syntax during the build, but do
+		not report warnings for missing comments.
 
+		CS1573: Parameter 'parameter' has no matching param tag in the XML comment for 'parameter' (but other parameters do)
+		CS1591: Missing XML comment for publicly visible type or member 'Type_or_Member'
+	  -->
+	  <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>

```

Comment:
Is this file missing? I know it is empty now, but as far as I can recall, we included an empty xml file in other similar PRs

Reply:
Yup, I missed it. Added now. Tysm!